Tournament_Calculator.py

```diff
@@ -0,0 +1,323 @@
+        'Second':{"Arakni":(sheet_obj.cell(row = (55), column = 3).value),"Azalea":(sheet_obj.cell(row = (55), column = 4).value),"Betsy":(sheet_obj.cell(row = (55), column = 5).value),"Bravo":(sheet_obj.cell(row = (55), column = 6).value),"DIO":(sheet_obj.cell(row = (55), column = 7).value),"Dash":(sheet_obj.cell(row = (55), column = 8).value),"Dorinthea":(sheet_obj.cell(row = (55), column = 9).value),"Fai":(sheet_obj.cell(row = (55), column = 10).value),"Kano":(sheet_obj.cell(row = (55), column = 11).value),"Kassai":(sheet_obj.cell(row = (55), column = 12).value),"Katsu":(sheet_obj.cell(row = (55), column = 13).value),"Kayo":(sheet_obj.cell(row = (55), column = 14).value),"Levia":(sheet_obj.cell(row = (55), column = 15).value),"Maxx":(sheet_obj.cell(row = (55), column = 16).value),"Olympia":(sheet_obj.cell(row = (55), column = 17).value),"Prism":(sheet_obj.cell(row = (55), column = 18).value),"Rhinar":(sheet_obj.cell(row = (55), column = 19).value),"Riptide":(sheet_obj.cell(row = (55), column = 20).value),"Boltyn":(sheet_obj.cell(row = (55), column = 21).value),"Teklovossen":(sheet_obj.cell(row = (55), column = 22).value),"Uzuri":(sheet_obj.cell(row = (55), column = 23).value),"Victor":(sheet_obj.cell(row = (55), column = 24).value),"Viserai":(sheet_obj.cell(row = (55), column = 25).value),"Vynnset":(sheet_obj.cell(row = (55), column = 26).value),"Enigma":(sheet_obj.cell(row = (55), column = 27).value),"Nuu":(sheet_obj.cell(row = (55), column = 28).value),"Zen":(sheet_obj.cell(row = (55), column = 29).value)}}}
+
+
+
+
+#Swiss Pairings Logic
+def generate_pairings(players, previous_pairings=None, results=None):
+    pairings = []
+    if previous_pairings is None:
+        previous_pairings = set()  # Initialize an empty set for previous pairings
+    if results is None:
+        results = {}  # Initialize an empty dictionary for results

```

Comment:
Is there a more specific / descriptive name than "results" that you could use? Still having a little trouble following this easily